File: assessments/object_relations/app/models/review.rb
require './restaurant.rb'

class Review
  attr_accessor :customer, :restaurant, :content

    @@all = []

    def initialize(content, restaurant, customer)
      @@all << self

      customer.reviews << self
      restaurant.reviews << self

      @restaurant = restaurant
      @customer = customer
      @content = content
    end
end

File: assessments/javascript/models/review.js
// # Restaurant#customers
// # Restaurant#reviews

const Review = class Review {

  static all() {
    Store.reviews
  }

  constructor(content, customer, restaurant) {
    this.content = content
    this.customer = customer
    this.restaurant = restaurant
    Store.reviews.push(this)
  }
  }
